src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -313,21 +323,155 @@ public static void main(final String[] args) {
         return 0;
     }
 
-    protected static GenotypesContext fixGenotypes(final GenotypesContext originals, final Map<Allele, Allele> reverseComplementAlleleMap) {
+    protected static Locatable getFlippedLocation(final VariantContext source, final LiftOver liftOver) {
+        Interval originalLocus = new Interval(source.getContig(), source.getStart() + 1, source.getEnd() + 1);
+        return liftOver.liftOver(originalLocus);
+    }
+
+    /**
+     * @param source            original variant context
+     * @param liftOver          the LiftOver object to use for flipping
+     * @param referenceSequence the reference sequence of the target
+     * @return a flipped variant-context.
+     */
+    protected static VariantContext flipIndel(final VariantContext source, final LiftOver liftOver, final ReferenceSequence referenceSequence) {
+        if (!source.isBiallelic()) return null;  //only supporting biallelic indels, for now.
+
+        final Locatable target = getFlippedLocation(source, liftOver);
+        if (target == null) return null;
+
+        final Map<Allele, Allele> reverseComplementAlleleMap = new HashMap<>(10);
+
+        reverseComplementAlleleMap.clear();
+        final List<Allele> alleles = new ArrayList<>();
+
+        for (final Allele oldAllele : source.getAlleles()) {
+            // target.getStart is 1-based, reference bases are 0-based
+            final String alleleWithoutHead = oldAllele.getBaseString().substring(1, oldAllele.length());
+            final char refBase = (char) referenceSequence.getBases()[target.getStart() - 1];
+            final Allele fixedAllele = Allele.create(refBase + SequenceUtil.reverseComplement(alleleWithoutHead), oldAllele.isReference());
```

Comment:
Question for you: is it worth confirming for deletions that the bases that were deleted in the source are the same ones being deleted in the target?  I.e. if bases "ACC" are deleted in the source then we'd expect their reverse complemented equivalents to be deleted in the target, or else we fail.

I'm not requiring this to be added as part of the review, but I would then want this described in the docs for the tool up top so that it's clear what it does and does not do.